### PR TITLE
Fix typo in custom metric name

### PR DESCRIPTION
**Pull request by tomknig**
I've spotted this (probably unintended) typo on https://www.tensorflow.org/guide/keras/train_and_evaluate

### CODE REVIEW

**Comment by tomknig** [comment]
@googlebot I signed it!
